### PR TITLE
♻️ Add YT.get props to be more realistic

### PR DESCRIPTION
**Pull request by castroCrea**
In the lib we miss the `YT.get`
image

### CODE REVIEW

File: types/youtube/index.d.ts [hunk 1]
```diff
@@ -697,6 +697,16 @@ declare namespace YT
...
+    export class get extends Player {
+      /**
+       * Get the instance of the Player class attached to an iframe.
+       *
+       * @param iframeId   Id of the iframe.
+       * @param options   Player options.
+       */
+      constructor(iframeId: string);
+    }
+
```
Comment (rbuckton) [inline types/youtube/index.d.ts:709]: I don't think `get` is a `class`. It looks like it's just a function that returns a registered player by its `iframe`'s id, so you shouldn't need to call `new`.

```suggestion
   /**
    * Get the instance of the Player class attached to an iframe.
    *
    * @param iframeId   Id of the iframe.
    */
    export function get(iframeId: string): Player | undefined;

```

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/youtube/index.d.ts (hunk 1 above)
Comment (castroCrea) [reply types/youtube/index.d.ts]: I made the change.

On my side, I always use it with a new and never got any error of this type
image

this is what I get when doing a new
image

Comment (jakebailey) [reply types/youtube/index.d.ts]: I just skimmed the YT iframe reference [here](https://developers.google.com/youtube/iframe_api_reference?hl=en) to see if I could figure this one out, but I don't see this function documented at all. Do you have a reference that shows this API?

The PR template asks for this, but you delete the template entirely.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Please address feedback from @jakebailey and give us detail documentation why this change needs to be made

**Comment by castroCrea** [comment]
The doc is not complet

[here](https://developers.google.com/youtube/iframe_api_reference) you can get the script to load the YT api from youtube 
So you can check the source of the [script](https://www.youtube.com/iframe_api) 

Inside you can get the `scriptURL` https://www.youtube.com//s//player//6a14c793//www-widgetapi.vflset//www-widgetapi.js

In there just to a cmd+F `YT.get` 
You will see the function

**Review by jakebailey — APPROVED** [review]
Thanks for the codepen; this is definitely a function.

**Pull request closed without merging**